### PR TITLE
ENV: Remove duplicate settings

### PR DESCRIPTION
**Pull request by GaryJones**
PR https://github.com/wecodemore/wpstarter/pull/16 to the `dev` branch.

### CODE REVIEW

File: wpstarter/templates/.env.example [hunk 1]
```diff
@@ -227,12 +227,10 @@ WP_HOME=
...
-#WP_HTTP_BLOCK_EXTERNAL
 
 
 #--------------------------------------------------------------------------------------------------#
 #                                          MISCELLANEOUS                                           #
 #--------------------------------------------------------------------------------------------------#
 #WP_MAIL_INTERVAL=300
-#WP_DEFAULT_THEME=twentyfifteen
\ No newline at end of file
+#WP_DEFAULT_THEME=twentyfifteen
```
Comment (GaryJones) [inline wpstarter/templates/.env.example]: Does this need bumping to the latest version too?

Comment (franz-josef-kaiser) [inline wpstarter/templates/.env.example]: You mean tagging a new version? Ping me tomorrow, then I'll add a new patch level version and fix the wrong v2.1.1 as well. Thanks in advance!

Comment (GaryJones) [inline wpstarter/templates/.env.example]: I meant, should the WPStarter define `twentysixteen` as the commented-out default theme, as presumably, that's what WP 4.4 defines as the default theme?

```php
#WP_DEFAULT_THEME=twentysixteen
```

Comment (Giuseppe-Mazzapica) [inline wpstarter/templates/.env.example]: @GaryJones the minimum required WP version for WP starter is 4.2, where twentysixteen is  not available yet. So this needs to stay twentyfifteen untill the min version is bumped up.

I am ready to merge this, if @franz-josef-kaiser is agree (I saw he self-assigned the issue) we can merge this like now.

**Pull request merged**